cli/azd/pkg/project/service_target_containerapp.go

```diff
@@ -86,26 +95,74 @@ func (at *containerAppTarget) Deploy(
 		return nil, err
 	}
 
-	containerAppOptions := containerapps.ContainerAppOptions{
-		ApiVersion: serviceConfig.ApiVersion,
+	// Default resource name and type
+	resourceName := targetResource.ResourceName()
+	aspireDeploymentType := azapi.AzureResourceTypeContainerApp
```

Comment:
this service_target was not aware of Aspire.  Are we duplicating Aspire deploying logic here and in service_target_dotnet_containerapp ?